js/apps/events-page/controllers/eventRegistrationCtrl.js

```diff
@@ -30,10 +30,8 @@
 				vm.postResult = postResult.data;
 
 				const data = vm.postResult.Data;
-				const hasConfirmationMessage = data &&
-                    Object.prototype.hasOwnProperty.call(data, 'ConfirmationMessage') &&
-                    data.ConfirmationMessage &&
-					data.ConfirmationMessage.length;
+				const hasConfirmationMessage = data && Object.prototype.hasOwnProperty.call(data, 'ConfirmationMessage') &&
+                    data.ConfirmationMessage && data.ConfirmationMessage.length;
```

Comment:
Not gonna lie... This somehow looks worse.  Is there a mix of tabs and spaces or something?

Reply:
@ascire yes something is up with my IDE